### PR TITLE
Added 9.1.0 beta info bug

### PR DESCRIPTION
**Pull request by aadharsh-rengarajan**
https://freshworks.freshrelease.com/ws/MP/tasks/MP-32395

### CODE REVIEW

File: path.bash.inc [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
 # Get the installation location of the fdk formula
+# Check for fdk
 installation_location=$(brew info fdk | grep Cellar | cut -d ' ' -f 1)
-export PATH="$installation_location/custom_node_location/bin:$PATH"
\ No newline at end of file
+if [[ -n "$installation_location" ]]; then
+    export PATH="$installation_location/custom_node_location/bin:$PATH"
+    echo "fdk found and added to PATH"
+else
+    echo "fdk not found"
```
Comment (nadeem-freshworks) [inline path.bash.inc]: Please add better comments here since its customer facing it should be proper message

File: path.bash.inc [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+fi
+
+# Check for fdk@9.1.0-beta
+installation_location_beta=$(brew info fdk@9.1.0-beta | grep Cellar | cut -d ' ' -f 1)
+if [[ -n "$installation_location_beta" ]]; then
+    export PATH="$installation_location_beta/custom_node_location/bin:$PATH"
+    echo "fdk@9.1.0-beta found and added to PATH"
+else
+    echo "fdk@9.1.0-beta not found"
```
Comment (nadeem-freshworks) [inline path.bash.inc]: same as above

**Review by nadeem-freshworks — CHANGES_REQUESTED** [review]
Did we tested the scenario where i have fdk 9 and fdk 9.1.0 both present in the system
Which one it will pick ?

**Comment by aadharsh-rengarajan** [comment]
> Did we tested the scenario where i have fdk 9 and fdk 9.1.0 both present in the system Which one it will pick ?

In that case it will pick the beta one

**Pull request merged**